src/debug.py

```diff
@@ -41,9 +42,12 @@
 import sys
 
 from six.moves import configparser
-
-import helper_startup
-import state
+try:
+    import helper_startup
+    import state
+except ImportError:
+    from . import helper_startup
```

Comment:
puth these lines together

Reply:
You don't need this at all. Just don't use debug in `sqlThread` and make it like other threads to use own logger, gotten in the class definition, look: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/network/threads.py#L9

I reverted it once already: b75585f2

Reply:
This PR is still a WIP, you don't need to look at it yet, except in some cases like these imports which are not my strength.

Reply:
I receive a notification from this PR, because it was not originally a draft, it was me, who made it draft.